kbfsgit/runner.go

```diff
@@ -251,8 +251,14 @@ func (r *runner) initRepoIfNeeded(ctx context.Context, forCmd string) (
 		}()
 	}
 
-	fs, _, err = libgit.GetOrCreateRepoAndID(
-		ctx, r.config, r.h, r.repo, r.uniqID)
+	// Only allow lazy creates for public and multi-user TLFs.
+	if r.h.Type() == tlf.Public || len(r.h.ResolvedWriters()) > 1 {
```

Comment:
What about unresolved writers? I should be able to make a git repository with `jzila,foo@twitter`.

Reply:
Good point, will fix.